packages/next/src/server/dev/hot-reloader-turbopack.ts

```diff
@@ -666,15 +667,21 @@ export async function createHotReloaderTurbopack(
           }
         }
 
-        const sync: SyncAction = {
-          action: HMR_ACTIONS_SENT_TO_BROWSER.SYNC,
-          errors,
-          warnings: [],
-          hash: '',
-          versionInfo,
-        }
+        ;(async function () {
+          if (!versionInfo) {
+            versionInfo = await versionInfoPromise
+          }
```

Comment:
```suggestion
          const versionInfo = await versionInfoPromise
```

This could be simplified, since once a Promise is resolved you can await it multiple times without it recomputing

Reply:
TIL thank you!